docs/params.md

```diff
@@ -15,7 +15,7 @@ There are two pieces to this:
 2. Read the params in your screen component: `this.props.navigation.getParam(paramName, defaultValue)`.
 
 > We recommend that the params you pass are JSON-serializable. That way, you'll be able to use [state persistence](state-persistence.html) and your screen components will have the right contract for implementing [deep linking](deep-linking.html).
-
+#TODO how about get Params? Maybe we can support it?
```

Comment:
I think now that we have optional chaining and nullish coalescing operator, we don't need this:

```js
const itemId = navigation.getParam('itemId', 'NO-ID');
```

Can be rewritten to:

```js
const itemId = route.params?.itemId ?? 'NO-ID';
```